test/integration/sanity/Sanity.e2e.test.js

```diff
@@ -17,10 +17,15 @@ describe('Preview Sanity', () => {
         cy.showPreview(token, fileId);
         cy.getPreviewPage(1);
         cy.contains('The Content Platform for Your Apps');
+        cy.get('.bp-thumbnail').each($ele => {
```

Comment:
Should this be moved to a helper, like `cy.hasThumbnails`, since it's used more than once?